### PR TITLE
fix(TryIt): prefer default value over example value

### PR DESCRIPTION
**Pull request by lottamus**
Fixes https://github.com/stoplightio/platform-internal/issues/8990

### CODE REVIEW

File: packages/elements-core/src/components/TryIt/TryIt.spec.tsx [hunk 1]
```diff
@@ -247,8 +247,8 @@ describe('TryIt', () => {
       // header param
 
       const accountIdField = screen.getByLabelText('account-id') as HTMLInputElement;
-      expect(accountIdField).toHaveValue('example id');
-      expect(accountIdField.placeholder).toBe('example: example id');
+      expect(accountIdField).toHaveValue('account-id-default');
+      expect(accountIdField.placeholder).toBe('example: account-id-default');
```
Comment (lottamus) [inline packages/elements-core/src/components/TryIt/TryIt.spec.tsx:251]: We could optionally update this to say `default` too 🤔 :

```suggestion
      expect(accountIdField.placeholder).toBe('default: account-id-default');
```

Comment (marbemac) [reply packages/elements-core/src/components/TryIt/TryIt.spec.tsx:251]: `defaults to: xyz`?

Comment (lottamus) [reply packages/elements-core/src/components/TryIt/TryIt.spec.tsx]: image

Comment (lottamus) [reply packages/elements-core/src/components/TryIt/TryIt.spec.tsx]: Done in e99e06fc78593974eeb543b53356732873599c90

**Review by marbemac — APPROVED** [review]

**Review by mnaumanali94 — APPROVED** [review]

**Pull request merged**